src/surfaces/drawerDeprecated/drawerDeprecated.jsx

```diff
@@ -290,11 +290,14 @@ class DrawerDeprecated extends React.Component {
         const animationEvent = this._transitionProps(this._drawerContainer);
 
         document.body.classList.add('drawer-animate-out');
-        this._drawerContainer.classList.add('drawer-animate-out');
-        this._drawerContainer.style.transform = this.props.position === 'right' ?
-            'translate(100%, 0)' :
-            'translate(-100%, 0)';
-        this._drawerContainer.addEventListener(animationEvent, this._onCloseAnimationComplete);
+
+        if (!!this._drawerContainer) {
```

Comment:
In my opinion, we should either allow this check for truthiness (without `!!`) or we should use something like `!isNil(this._drawerContainer)`.

Reply:
`!!` removed

Reply:
Looks good.  Resolving.